src/app/metadata/features/metadata-browser/metadata-browser.component.spec.ts

```diff
@@ -6,15 +6,30 @@
 
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
+import { signal } from '@angular/core';
+
+import { searchResults } from '@app/../mocks/data';
+import { MetadataSearchService } from '@app/metadata/services/metadataSearch.service';
 import { MetadataBrowserComponent } from './metadata-browser.component';
 
+/**
+ * Mock the metadata service as needed for the global stats
+ */
+class MockMetadataSearchService {
+  searchResults = signal(searchResults);
+  searchResultsError = signal(undefined);
```

Comment:
the mock service is missing the load() method

Reply:
Done 👍